doc/admin-guide/files/records.config.en.rst

```diff
@@ -3100,6 +3100,28 @@ SSL Termination
 
    Configures the cipher_suite which |TS| will use for SSL connections to origin or next hop.
 
+.. ts:cv:: CONFIG proxy.config.ssl.server.groups_list STRING <See notes>
+
+   Configures the list of supported groups provided by OpenSSL which
+   |TS| will be used to determine the set of shared groups. The value
+   is a colon separated list of group NIDs or names, for example
+   "P-521:P-384:P-256".
+
+   The current default value with OpenSSL is:
+
+   X25519:P-256:X448:P-521:P-384
+
+   This configuration works with OpenSSL v1.1.1 and above.
+
+.. ts:cv:: CONFIG proxy.config.ssl.client.groups_list STRING <See notes under proxy.config.ssl.server.groups_list.>
+
+   Configures the list of supported groups provided by OpenSSL which
+   |TS| will use for the "key_share" and "supported groups" extention
+   of TLSv1.3 connections. The value is a colon separated list of
+   group NIDs or names, for example "P-521:P-384:P-256".
+
+   This configuration works with OpenSSL v1.1.1 and above.
```

Comment:
Please add a link to the openssl documentation.

Reply:
Do you mean man page of `SSL_CTX_set1_groups_list` ?
https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set1_groups_list.html

or more general description? 
https://wiki.openssl.org/index.php/TLS1.3#Groups